cli/command/image/push.go

```diff
@@ -33,6 +36,7 @@ func NewPushCommand(dockerCli command.Cli) *cobra.Command {
 	}
 
 	flags := cmd.Flags()
+	flags.BoolVarP(&opts.all, "all-tags", "a", false, "Push all tagged images in the repository")
```

Comment:
Do we really need `-a` ? I think that `all-tags` is short enough, and is not the main use case, not even a common use case  🤔

Reply:
Yeah, it's consistent with `docker pull` which also has both